go.mod

```diff
@@ -1,10 +1,10 @@
 module github.com/gobuffalo/mw-csrf
 
-go 1.13
+go 1.16
 
 require (
-	github.com/gobuffalo/buffalo v0.15.4
-	github.com/gobuffalo/envy v1.8.1
-	github.com/gobuffalo/httptest v1.4.1
-	github.com/stretchr/testify v1.4.0
+	github.com/gobuffalo/buffalo v0.18.7
```

Comment:
Check `gobuffalo/cli` or `gobuffalo/buffalo` for a dependabot config. This should really be done automatically in the future.

Reply:
let's discuss that in the org-wide discussion. Actually, I have not enough permissions for all repos.

(personally, I love automated alerts but don't like commit history that is full of robot commits and release notes since it makes it hard to find interesting information)